iocage_lib/ioc_create.py

```diff
@@ -628,6 +633,9 @@ def create_config(self, jail_uuid, release):
             jail_props.update(d_conf)
             jail_props['CONFIG_TYPE'] = 'THICK'
 
+        if source_template is not None:
```

Comment:
Should we perhaps take care of this prop as well as we discussed ?

Reply:
That's a different PR to rename that, unless you're talking about something else?

Reply:
Oh, I think I misunderstood you. Are you saying we should take care of that use case in another PR ?

Reply:
Yep :)